servlet/src/main/scala/org/http4s/servlet/AsyncHttp4sServlet.scala

```diff
@@ -118,13 +106,11 @@ class AsyncHttp4sServlet[F[_]](
 object AsyncHttp4sServlet {
   def apply[F[_]: ConcurrentEffect](
       service: HttpRoutes[F],
-      asyncTimeout: Duration = Duration.Inf,
-      executionContext: ExecutionContext = ExecutionContext.global): AsyncHttp4sServlet[F] =
+      asyncTimeout: Duration = Duration.Inf): AsyncHttp4sServlet[F] =
     new AsyncHttp4sServlet[F](
       service,
       asyncTimeout,
-      executionContext,
-      BlockingServletIo[F](DefaultChunkSize),
+      NonBlockingServletIo[F](DefaultChunkSize),
```

Comment:
This is going to hurt benchmarks, but help in scalability.  We might gain a fair bit of it back losing that other shift.  And it's still pluggable.

Reply:
I don't think we bench servlets so it's all good.

Reply:
We should, to monitor for regressions and let people make informed choices. We just never automated anything.